app/views/optIn/BeforeYouStart.scala.html

```diff
@@ -0,0 +1,76 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.html.components._
+@import views.html.layouts.unifiedLayout
+@import models.incomeSourceDetails.TaxYear
+@import views.html.helpers.injected.ErrorSummary
+
+@this(
+        mainTemplate: unifiedLayout,
+        formWithCSRF: FormWithCSRF,
+        govukRadios: GovukRadios,
+        govukButton: GovukButton,
+        errorSummary: ErrorSummary,
+        h1: h1,
+        h2: h2,
+        p: p,
+        link: link,
+        hr: hr,
+        govukTable: GovukTable,
+        govukInsetText: GovukInsetText,
+)
+
+
+@(isAgent: Boolean = false)(implicit messages: Messages, user: auth.MtdItUser[_])
+
+@mainTemplate(
+    pageTitle = messages("optIn.beforeYouStart.heading"),
+    backUrl = Some("#"),
+    isAgent = isAgent,
+    btaNavPartial = user.btaNavPartial,
+    useFallbackBackLink = true
+) {
+
+    @h1(id = Some("heading"), msg = messages("optIn.beforeYouStart.heading"), classes = "govuk-heading-xl")
+
+    <div id="content" class="govuk-!-margin-bottom-7">
+        @p(id = Some("desc1")){@messages("optIn.beforeYouStart.desc1")}
+        @p(id = Some("desc2"), classes = "govuk-inset-text govuk-!-margin-bottom-6"){
+            @messages("optIn.beforeYouStart.desc2")
+        }
+        @h2(optId = Some("reportQuarterly"), msg = messages("optIn.beforeYouStart.reportQuarterly"), classes = "govuk-heading-m")
+        @p(id = Some("reportQuarterly-text")){@messages("optIn.beforeYouStart.reportQuarterly.text")}
+        <ul id="reportQuarterly-list" class="govuk-list govuk-list--bullet">
+            <li>
+                @messages("optIn.beforeYouStart.reportQuarterly.text.p1")
+            </li>
+            <li>
+                @messages("optIn.beforeYouStart.reportQuarterly.text.p2")
+            </li>
+        </ul>
+        @h2(optId = Some("voluntaryStatus"), msg = messages("optIn.beforeYouStart.voluntaryStatus"), classes = "govuk-heading-m")
+        @p(id = Some("voluntaryStatus-text")){@messages("optIn.beforeYouStart.voluntaryStatus.text")}
+    </div>
+
+    <div class="govuk-button-group">
+        @govukButton(Button(
+        content = Text(messages("optIn.beforeYouStart.button.start")),
+        attributes = Map("id" -> "start-button")
```

Comment:
Just checking: will the button link be added when MISUV-7943 is done or should the link be present now to the non-existent page?